privateSBOMExchange/src/scripts/Redaction.py

```diff
@@ -260,7 +65,7 @@ def main():
 
     
 
-    #using tree stucture, store sbom as tree in sqlalchamey db
+    #using tree stucture, store sbom as tree in sqlalchemy db
     t1=try_tree()
```

Comment:
I understand that this is a work in progress, but we really shouldn't be mixing testing with core system APIs. The Redaction CLI should do only one thing: take an SBOM and output a redacted version.